### PR TITLE
Turning on trivy scanning for other containers built in pipeline

### PR DESCRIPTION
**Pull request by andytinkham**
Signed-off-by: Andy Tinkham <andy.tinkham@cyberark.com>

#### What does this PR do (include background context, if relevant)?
Adds scanning for the secretless-broker-quickstart and secretless-broker-redhat containers that are built as part of the pipeline.

#### What ticket does this PR close?
N/A
#### What is the status of the manual tests?
Have you run the following manual tests to verify existing functionality continues to function as expected?
- [ ] Manually tested [Keychain provider](https://github.com/cyberark/secretless-broker/tree/master/test/providers/keychain)
- [ ] Manually run [Kubernetes-Conjur demo](https://github.com/conjurdemos/kubernetes-conjur-demo) with a local Secretless Broker image build of your branch

If this feature does not have any/sufficent automated tests, have you created/updated a folder in `test/manual` that includes:
- [ ] An updated README with instructions on how to manually test this feature
- [ ] Utility `start` and `stop` scripts to spin up and tear down the test environments
- [ ] A `test` script to run some basic manual tests (optional; if does not exist, the README should have detailed instructions)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -35,12 +35,27 @@ pipeline {
...
+        }
+
+        stage('Scan Secretless RedHat') {
+          steps {
+            script {
+              TAG = sh(returnStdout: true, script: '. bin/build_utils && full_version_tag')
+            }
+            scanAndReport("secretless-broker-redhat:${TAG}", "NONE")
+          }
+        }
```
Comment (izgeri) [inline Jenkinsfile]: is there any reason not to run these scans in parallel? if not, I have a strong preference for parallelizing this stage.

Comment (andytinkham) [reply Jenkinsfile]: They're already in parallel - the original Jenkinsfile parallelized the first trivy scan and the Go sec run.

File: Jenkinsfile [hunk 2]
```diff
@@ -35,12 +35,27 @@ pipeline {
 
     stage('Scan Secretless') {
       parallel {
-        stage('Scan Secretless Image') {
+        stage('Scan Secretless Images') {
```
Comment (izgeri) [inline Jenkinsfile]:
```suggestion
        stage('Scan Secretless Image') {
```

Comment (andytinkham) [reply Jenkinsfile]: Agreed - not sure what I was thinking there. Thanks!

**Pull request merged**